### ISSUE TITLE
Authentication : Login 

### ISSUE THREAD

**Issue by ditikrushna** [ditikrushna/Medium-like-Blog-Backend#4]
**API End Point :** `POST /apis/users/login`

**Example request body:**

    {
      "user":{
        "email": "arun@arun.arun",
        "password": "aaabbbaaabbba"
      }
    }

No authentication required, returns a **User** 

 **JSON Objects returned by API**
    
    {
      "user": {
        "email": "arun@arun.arun",
        "token": "jwt.token.here",
        "username": "arun",
        "bio": "I work at xyz",
        "image": null
      }
    }

Required fields:  `email`,  `password`

**Comment by cecil-m** [comment]
Hi, i would like to pick up this issue.
Probably use passport.js package, unless you have any alternative solution

**Comment by ditikrushna** [comment]
Hi @cecil-m , Thanks for your interest . Right now , I am creating the roadmap ,structure and documentation of the project .  
Once I am done with the all these stuff you can start working on it .

**Comment by ditikrushna** [comment]
@cecil-m yeah you can move ahead with [passport js](http://www.passportjs.org/)
